hazelcast-sql/src/test/java/com/hazelcast/jet/sql/SqlOrderByTest.java

```diff
@@ -137,7 +139,7 @@ public void before() {
         members = createHazelcastInstances(memberConfig(), membersCount);
 
         if (isPortable()) {
-            createMapping(members[0], mapName(), PORTABLE_FACTORY_ID, PORTABLE_VALUE_CLASS_ID, 0, PORTABLE_FACTORY_ID, PORTABLE_VALUE_CLASS_ID, 0);
+            createMapping(members[0], mapName(), PORTABLE_FACTORY_ID, PORTABLE_KEY_CLASS_ID, 0, PORTABLE_FACTORY_ID, PORTABLE_VALUE_CLASS_ID, 0);
```

Comment:
portable tests were never working here, maybe should be completely removed (in separate PR)